modules/nextflow/src/main/groovy/nextflow/conda/CondaCache.groovy

```diff
@@ -70,6 +72,10 @@ class CondaCache {
 
     @PackageScope Path getConfigCacheDir0() { configCacheDir0 }
 
+    @PackageScope String binaryName() {
+        useMamba ? "mamba" : "conda"
+    }
+
```

Comment:
Renaming this to `getBinaryName` it becomes a *synthetic* attribute that you can access as `binaryName` instead of `binaryName()`

Reply:
Interesting - thanks for the tip 😊